segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/FileSystemIntegrationTest.java

```diff
@@ -60,6 +60,8 @@ protected ServiceBuilder createBuilder(ServiceBuilderConfig.Builder configBuilde
                 .withStorageFactory(setup -> useChunkedSegmentStorage ?
                         new FileSystemSimpleStorageFactory(ChunkedSegmentStorageConfig.DEFAULT_CONFIG.toBuilder()
                                 .journalSnapshotInfoUpdateFrequency(Duration.ofMillis(10))
+                                .garbageCollectionDelay(Duration.ofMillis(10))
```

Comment:
I think we should do this for every LTS integration test, not just for File System.

Reply:
done